src/frontend/apps/impress/src/features/left-panel/components/LeftPanelFavorites.tsx

```diff
@@ -53,23 +50,36 @@ export const LeftPanelFavorites = () => {
         >
           {invitations.map((doc) => (
             <Box
+              $direction="row"
+              $align="center"
+              $justify="space-between"
               $css={css`
                 padding: ${spacing['2xs']};
                 border-radius: 4px;
+                .pinned-actions {
+                  opacity: 0;
+                }
                 &:hover {
                   cursor: pointer;
+
                   background-color: var(--c--theme--colors--greyscale-100);
+                  .pinned-actions {
+                    opacity: 1;
+                  }
                 }
               `}
               key={doc.id}
             >
               <StyledLink href={`/docs/${doc.id}`}>
                 <SimpleDocItem showAccesses doc={doc} />
               </StyledLink>
+              <div className="pinned-actions">
+                <DocsGridActions doc={doc} />
```

Comment:
The Share button does not open the modal.
![image](https://github.com/user-attachments/assets/885055e7-ebfa-4691-9564-be75a3d6e596)